pkg/datarepair/repairer/config.go

```diff
@@ -19,6 +27,15 @@ type Config struct {
 	QueueAddress string        `help:"data repair queue address" default:"redis://127.0.0.1:6378?db=1&password=abc123"`
 	MaxRepair    int           `help:"maximum segments that can be repaired concurrently" default:"100"`
 	Interval     time.Duration `help:"how frequently checker should audit segments" default:"3600s"`
+	storeConfig
+}
+
+type storeConfig struct {
```

Comment:
Why is this a new struct? Can't we reuse the `ClientConfig`?

Reply:
Done